### PR TITLE
Sidebar visuals

### LINKED ISSUE #945: (1.1.1) Visual Design: Main menu updates
screen shot 2017-12-06 at 05 36 36

1. Menu items are currently off from standard positioning. I've refined the whole positioning logic a bit, see new specs below. Also included a wallets icon, this is mostly for being a hint as well helps to align the elements.

2. Incorporated rescan into the main menu. The Icon should work as a rescan button. Upon click, displays the progress bar and a cancel icon on the right side. 

Visual design + specs: https://xd.adobe.com/view/3f97bc61-1369-42eb-ab9e-d8725178d9f4/ 

New Icons: [decrediton - menu - icons 061217.zip](https://github.com/decred/decrediton/files/1533784/decrediton.-.menu.-.icons.061217.zip)

### PR DESCRIPTION
**Pull request by matheusd**
Fix #945 

I've removed the rescan button from the home screen, but I'm unsure whether only the button on the sidebar is enough (as it may be somewhat hard to notice). I'll update the issue with previews, so linnutee can check out.

### CODE REVIEW

**Comment by alexlyp** [comment]
Maybe just a visual distortion/optical illusion, but it looks like the rescan icon and "Last block..." are a pixel or 2 to the left of the wallet icon and Total Balance above.

**Comment by alexlyp** [comment]
With the tooltip so close to the bottom we need slight tweak:

https://github.com/decred/decrediton/blob/master/app/components/shared/Tooltip.js#L12

Should be - 5 to give a buffer near the edge.

File: app/components/indicators/RescanProgress.js [hunk 1]
```diff
@@ -10,15 +10,27 @@ const RescanProgress = ({
...
-      max={1}
-      value={rescanCurrentBlock/rescanEndBlock}
+    <div className="rescan-progress-indicator">
+      <LinearProgress
+        mode="determinate"
+        min={0}
+        max={1}
+        value={rescanCurrentBlock/rescanEndBlock}
+        color="#2ed8a3"
+        margin
```
Comment (alexlyp) [inline app/components/indicators/RescanProgress.js]: this is throwing a warn for me

**Comment by matheusd** [comment]
Tried as much as possible to align things, take a look below. The icons are very different (hollow vs filled) and the aliasing of the font sometimes makes things look a bit off. 


image
image

**Comment by alexlyp** [comment]
Yah looks good now.

**Pull request merged**